Packs/CommonScripts/Scripts/GetEndpointData/GetEndpointData.py

```diff
@@ -0,0 +1,719 @@
+        demisto.debug(f'run_command::{endpoints=}')
+        if command.post_processing:
+            endpoints = command.post_processing(endpoints, endpoint_args)
+
+        return hr, endpoints
+
+    def is_command_runnable(self, command: MappedCommand, args: dict[str, Any]) -> bool:
+        """
+        Executes a command if it is available and returns the results.
+
+        Args:
+            command (MappedCommand): An instance of MappedCommand that contains the command information.
+            args (Dict[str, Any]): A dictionary containing the specific arguments for the command.
+        Returns:
+            Tuple[List, str, List[CommandResults]]: A tuple containing the command outputs and any relevant results.
+        """
+        demisto.debug(f'is_command_runnable:: {command=}, args={args}')
+
+        if not self.module_manager.is_brand_available(command):
+            demisto.debug(f'Skipping command "{command.name}" since the brand {command.brand} is not available.')
+            return False
+
+        if not args.values() and command.name not in self.arg_free_commands:
+            demisto.debug(f'Skipping command "{command.name}" since the provided arguments does not match the command.')
+            return False
+
+        return True
+
+    @staticmethod
+    def run_execute_command(command: MappedCommand, args: dict[str, Any]) -> list[dict[str, Any]]:
+        """
+        Executes a command if the brand is available and returns the results.
+
+        Args:
+            command (MappedCommand): An instance of MappedCommand that contains the command information.
+            args (Dict[str, Any]): A dictionary containing the specific arguments for the command.
+        Returns:
+            Dict[str, Any]: A dictionary containing the command and its results.
+        """
+
+        return to_list(demisto.executeCommand(command.name, args))
+
+    @staticmethod
+    def get_commands_outputs(command: str, results: list[dict[str, Any]], args: dict[str, Any]) -> tuple[list, str, list]:
+        """
+        Processes the results of a previously executed command and extracts relevant outputs.
+
+        Args:
+            command (MappedCommand): An instance of MappedCommand that contains the command information.
+            results (Dict[str, Any]): A dictionary containing the command results.
+            args (Dict[str, Any]): A dictionary containing the specific arguments for the command.
+
+        Returns:
+            Tuple[List[Dict[str, Any]], str, List[Dict[str, Any]]]:
+                A tuple containing:
+                - A list of command context outputs.
+                - A human-readable string of the results.
+                - A list of command error outputs.
+        """
+
+        command_context_outputs = []
+        human_readable_outputs = []
+        command_error_outputs = []
+        demisto.debug(f'get_commands_outputs for command "{command}" with result {results}')
```

Comment:
I think it's a good idea to at least keep some of the logs as this script is hard to debug using IDE since we're using `execute_command`. It's quite a complex script with a lot of variability in the inputs and outputs and debugging it without any logs will be a nightmare.